.github/workflows/validation-jobs.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         run: ANDROID_NDK_ROOT=$ANDROID_NDK_LATEST_HOME cargo apk build --package bevy_mobile_example
 
   run-examples-linux-vulkan:
-    if: ${{ github.event_name == 'merge_group' }}
+    if: ${{ github.event_name != 'pull_request' }}
```

Comment:
Could you add a comment on why this is different, since all of the other jobs in this file uses `== 'merge_group'`?